sage-ios/SAGE/SAGE/ProfileOperations.swift

```diff
@@ -46,18 +46,31 @@ class ProfileOperations: NSObject {
         case .TwoUnit: hours = 2
         default: hours = 0
         }
-        
-        let params = ["user":
-            [
-                UserConstants.kFirstName: user.firstName!,
-                UserConstants.kLastName: user.lastName!,
-                UserConstants.kEmail: user.email!,
-                UserConstants.kLevel: hours,
-                UserConstants.kSchoolID: user.school!.id,
-                UserConstants.kCurrentPassword: password,
-                UserConstants.kPhotoData: photoData
+        var params: [String: [String: AnyObject]]
+        if let photoData = photoData {
```

Comment:
just add the photo data param to the user json object right? no need to copy and paste all this